### PR TITLE
Update reddit.py

### PR DESCRIPTION
**Pull request by yaba12**

### CODE REVIEW

File: reddit.py [hunk 1]
```diff
@@ -14,45 +18,55 @@
...
+	def getTitle(self):
+		reddit = getReddit()
+		subreddit = reddit.subreddit(self.nameOfSubreddit)
+		returnList = []
+		for submission in subreddit.hot(limit=self.limits):
+	   		returnList.append(submission.title)
+		return returnList
+
+	#gets the score. Returns list of scores.
+	def getScore(self):
```
Comment (Joshuad2uiuc) [inline reddit.py]: The reddit = getReddit() function is replicated multiple times. You can call it in __init__ and make it a class attribute.

Comment (Joshuad2uiuc) [reply reddit.py]: Also can do the same with the subreddit name

**Comment by Joshuad2uiuc** [comment]
-1

**Comment by aditya0099** [comment]
+1

**Pull request closed without merging**